Sources/XCRemoteCache/Config/XCRemoteCacheConfig.swift

```diff
@@ -129,6 +129,8 @@ public struct XCRemoteCacheConfig: Encodable {
     /// `/CoolLibrary/main.swift`will be represented as `$(COOL_LIBRARY)/main.swift`).
     /// Warning: remapping order is not-deterministic so avoid remappings with multiple matchings.
     var outOfBandMappings: [String: String] = [:]
+    /// Option to enable/disable TrustManager
```

Comment:
`TrustManager` might be a big obscure for not advanced user. How about:
 ```suggestion
    /// If true, SSL certificate validation is disabled  
```